cmd/kubeadm/app/util/system/kernel_validator_test.go

```diff
@@ -207,7 +207,8 @@ CONFIG_3=n`
 	v := &KernelValidator{
 		Reporter: DefaultReporter,
 	}
-	got, err := v.parseKernelConfig(bytes.NewReader([]byte(config)))
+	got, warn, err := v.parseKernelConfig(bytes.NewReader([]byte(config)))
```

Comment:
`parseKernelConfig` now returns just a couple of results (`got` and `err`), so this test won't compile.